website/content/en/highlights/2022-02-08-0-20-0-upgrade-guide.md

```diff
@@ -2,7 +2,7 @@
 date: "2022-02-08"
 title: "0.20 Upgrade Guide"
 description: "An upgrade guide that addresses breaking changes in 0.20.0"
-authors: ["topbz"]
+authors: ["topbz","spencergilbert"]
```

Comment:
Is `topbz` right and not `tobz`??

Reply:
Pfff, if @tobz wants to misspell his handle that's his choice...

Reply:
🤣 

I wouldn't _mind_ if you wanted to fix it for me. :)